modules/developers-guide/partials/curl_cql_directive.adoc

```diff
@@ -0,0 +1,10 @@
+.Getting directive information
```

Comment:
Just checking, is this supposed to start with a period?

Reply:
Yes, I think I actually do want that. Makes it a "tip" box.